### ISSUE TITLE
"Hulk (2021)" is called "Hulk (2022)" in ComicVine

### ISSUE THREAD

**Issue by OddBloke** [DieselTech/CBL-ReadingLists#58]
As the first comic was released in 2022, ComicVine calls the series "Hulk (2022)" instead of what the rest of the world (including [Marvel](https://www.marvel.com/comics/series/31374/hulk_2021_-_present) themselves) call it, "Hulk (2021)". I proposed a fix for this, but it was rejected: https://comicvine.gamespot.com/wiki/moderation/1650652/

I don't know if someone else might have better luck with getting CV fixed (this was my first ever edit), but at the moment the CBL files refer the 2021 volume which anyone using CV for metadata won't have (instead having a 2022 volume):

```
Marvel/Master Reading Order/CBRO/[Marvel] Marvel Master Reading Order Part #12 (WEB-CBRO).cbl:<Book Series="Hulk" Number="1" Volume="2021" Year="2022">
Marvel/Master Reading Order/CBRO/[Marvel] Marvel Master Reading Order Part #12 (WEB-CBRO).cbl:<Book Series="Hulk" Number="2" Volume="2021" Year="2022">
Marvel/Master Reading Order/CBRO/[Marvel] Marvel Master Reading Order Part #12 (WEB-CBRO).cbl:<Book Series="Hulk" Number="3" Volume="2021" Year="2022">
Marvel/Master Reading Order/CBRO/[Marvel] Marvel Master Reading Order Part #12 (WEB-CBRO).cbl:<Book Series="Hulk" Number="4" Volume="2021" Year="2022">
Marvel/Master Reading Order/CBRO/[Marvel] Marvel Master Reading Order Part #12 (WEB-CBRO).cbl:<Book Series="Hulk" Number="5" Volume="2021" Year="2022">
Marvel/Master Reading Order/CBRO/[Marvel] Marvel Master Reading Order Part #12 (WEB-CBRO).cbl:<Book Series="Hulk" Number="6" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 25.cbl:<Book Series="Hulk" Number="6" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="1" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="2" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="3" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="4" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="5" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="7" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Core/[Marvel] CMRO Core Reading Order-Part 24.cbl:<Book Series="Hulk" Number="8" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 39.cbl:<Book Series="Hulk" Number="5" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 39.cbl:<Book Series="Hulk" Number="6" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 39.cbl:<Book Series="Hulk" Number="7" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 39.cbl:<Book Series="Hulk" Number="8" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 39.cbl:<Book Series="Hulk" Number="9" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="1" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="2" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="3" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="4" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="5" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="6" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="7" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="8" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Main/[Marvel] CMRO Main Reading Order-Part 29.cbl:<Book Series="Hulk" Number="9" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 38.cbl:<Book Series="Hulk" Number="1" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 38.cbl:<Book Series="Hulk" Number="2" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 38.cbl:<Book Series="Hulk" Number="3" Volume="2021" Year="2022">
Marvel/Master Reading Order/CMRO/Expanded/[Marvel] CMRO Expanded Reading Order-Part 38.cbl:<Book Series="Hulk" Number="4" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="1" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="2" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="3" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="4" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="5" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="6" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="7" Volume="2021" Year="2022">
Marvel/Master Reading Order/Marvel Guides/2021-2023 Timeless/Part 1 Once and Future/[Marvel] 2021-2023 Part 1.4 Banner of War (MG).cbl:        <Book Series="Hulk" Number="8" Volume="2021" Year="2022">
```

**Comment by DieselTech** [comment]
I might sit on this one yet. I don't agree with CV that data comes from cover date when they haven't gone by that for the vast majority of their releases. 

I don't want to start publishing incorrect info just because 1 source of it chooses to have it listed wrong. We'll have to talk with some people and figure out the path to take when issues like this come up.

**Comment by OddBloke** [comment]
Yeah, 100% agreed.

I started writing some scripting locally to post-process these lists (before realising that most of my problems were due to using volume-number volume tagging instead of volume-year volume tagging and fixing the problem at its source): a potential option would be to have a GitHub Action which does something similar: transforms the repo's files into "CV-compatible" CBLs that are published for download $somewhere (GH Pages probably being easiest).

**Comment by OddBloke** [comment]
X-Force (2019) has a similar issue, incidentally: CV is at odds with the rest of the world by calling it "X-Force (2020)". Shall I fire off a separate issue, or do we want a single "CV volume naming mismatches" issue for now?

**Comment by DieselTech** [comment]
Yeah that specific x-force one is even worse because they published it as 2019 for 3 years before changing it to 2020. So anyone who added the series during the initial time has it as 2019. 

I think a single thread for now to track them might be best. If it gets too unwieldy then we can break it down more.

**Comment by flips22** [comment]
The struggle is that comic tagger also uses CV.  There isn't another option out there that I know of that uses what the rest of the world calls it. So if we don't bend the knee to CV then the CBLs are essentially broken.

**Comment by DieselTech** [comment]
> The struggle is that comic tagger also uses CV. There isn't another option out there that I know of that uses what the rest of the world calls it. So if we don't bend the knee to CV then the CBLs are essentially broken.

Not anymore they don't. https://github.com/comictagger/comictagger/issues/501

That is partly why I'm not ready to bend the knee. The fight for the iron throne is still in the air.

**Comment by DieselTech** [comment]
It looks like as of 9/1/23 ComicVine unblocked ComicTagger. I think for now we have to change the tags to conform, but I'll open a new mismatch thread going forward.
